service/kvmconfig/v2/resource/configmap/delete.go

```diff
@@ -10,11 +10,11 @@ import (
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"github.com/giantswarm/kvm-operator/service/keyv2"
+	"github.com/giantswarm/kvm-operator/service/kvmconfig/v2/key"
```

Comment:
This here should be v3.

Reply:
Why?

Reply:
I thought this is the v3 configmap. See my other comments below. The diff is not helpful. Maybe I just got confused.